### DISCUSSION TITLE
Is there a way to enlarge (or decrease/resize) a canvas AFTER a project's created?

### DISCUSSION THREAD

**Discussion by Pitchblack70** [keiwando/toonsquid#405] [Q&A]
If so, I have yet to see where to do it. Heeellllp! :P

**Comment by Cat-Butler** [comment]
Video editing app used on the output ? (Or, if a single frame, photo editing app?) 😅

However, more seriously, if you select the Background layer in the timeline, you can open up the Properties inspector, and turn the Infinite Background switch to ON.

You can then draw outside the original canvas area, with some limitations: vector brushes can draw anywhere, all pixel layers stay the same original size but specific pixel layers can be moved (and scaled) from their original position if you want to extend your drawing area - just be aware that you are painting on separate layers, so one swipe won't go across both pixel areas.

Use a camera layer to define the output size, which can be quite different to the original canvas size.

Note: Any non-vector resizing, either scaling pixel layers inside ToonSquid, or scaling rendered output in a video/photo editor, potentially risks loss of quality.

**Comment by keiwando** [comment]
https://toonsquid.com/handbook/guides/faq/#crop

**Reply by Pitchblack70** [reply]
Thank you!!!  Thank you for creating Toonsquid, too. I know it's a one person operation, but it's *already* IMHO the best animation program out there for the iPad... with both a reasonable learning curve and affordable!  :) :) :)

**Reply by keiwando** [reply]
Thank you :)
